recipes/ri/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "3.6" %}
+
+package:
+  name: ri
+  version: {{ version }}
+
+source:
+  url: https://github.com/InfOmics/RI/archive/{{ version }}.zip
+  sha256: 5168d47f3882a526be9276b57684b8c2d234ede80e42865f0039e6a5a432c254
+  patches:
+    - makefile.patch
+    
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - which ri36
```

Comment:
Is there a help command you can run?

Reply:
No - the command `ri36` itself prints a list with options, but for some reason it fails the test.